src/wp-includes/media.php

```diff
@@ -2043,13 +2041,17 @@ function wp_img_tag_add_loading_optimization_attrs( $image, $context ) {
  * to remove the attribute or set it to another accepted value.
  *
  * @since 6.1.0
+ * @deprecated 6.4.0 Use wp_get_loading_optimization_attributes()
```

Comment:
The closest replacement function isn't `wp_get_loading_optimization_attributes()`, but `wp_img_tag_add_loading_optimization_attrs()`.
```suggestion
 * @deprecated 6.4.0 Use wp_img_tag_add_loading_optimization_attrs() instead.
```